components/tag-list/tag-list-item-mixin.js

```diff
@@ -17,41 +19,38 @@ export const TagListItemMixin = superclass => class extends superclass {
 		return [labelStyles, css`
 			:host {
 				display: grid;
+				max-width: 100%;
 				outline: none;
 			}
 			:host([hidden]) {
 				display: none;
 			}
-			.tag-list-item-content {
-				height: 1rem;
-				margin: auto;
-				min-width: 0;
-				overflow: hidden;
-				text-overflow: ellipsis;
-				white-space: nowrap;
-			}
 			.tag-list-item-container {
 				background-color: var(--d2l-color-regolith);
 				border-radius: 6px;
 				box-shadow: inset 0 0 0 1px var(--d2l-color-gypsum), 0 2px 4px rgba(0, 0, 0, 0.03);
 				box-sizing: border-box;
 				color: var(--d2l-color-ferrite);
 				cursor: pointer;
-				display: flex;
+				max-width: 320px;
 				min-width: 0;
+				outline: none;
+				overflow: hidden;
 				padding: 0.25rem 0.6rem;
+				text-overflow: ellipsis;
 				transition: background-color 0.2s ease-out, box-shadow 0.2s ease-out;
+				white-space: nowrap;
+			}
+			.tag-list-item-container:focus {
```

Comment:
If a tag is focused... when hovered, the outline disappears.  Is that expected?

Reply:
Nope! Will fix, thanks for catching